tools/cwtool/CwTool.hs

```diff
@@ -81,6 +82,10 @@ topLevelCommands =
       "test-miner"
       "Test an external miner"
       TestMiner.main
+  , CommandSpec
+      "db-checksum"
+      "Generate a checksum of all the checkpointer database tables in a block (or range of blocks)"
```

Comment:
We need to decide what we want. It's either going to be

  - "Give me all the db row state between blocks a and b"
  - "Give me all the db row state up to block b"

Originally I meant the latter.